interface/patient_file/summary/stats.php

```diff
@@ -199,15 +199,19 @@ function load_location( location ) {
                 $rowclass = "noreturn";
             }
 
+
+
             echo " <tr class='text $rowclass;'>\n";
 
             //turn allergies red and bold and show the reaction (if exist)
             if ($key == "allergy") {
                 $reaction = "";
                 if (!empty($row['reaction'])) {
-                    $reaction = " (" . $row['reaction'] . ")";
+                    $reaction= " (" .xlt(ucfirst(str_replace("_"," ",$row['reaction']))) . ")";
```

Comment:
Isn't the reaction a list from list_options. If so, then should just run through the getListItemTitle() in the library/options.inc.php library.
```
$reaction = " (" . getListItemTitle($row['reaction']) . ")";
```

Reply:
o.k. changed it to : 
           $reaction= " (" .getListItemTitle("reaction",$row['reaction']) . ")";